docs/data/data-grid/rows/rows.md

```diff
@@ -279,28 +279,71 @@ For now, row reordering is disabled if sorting is applied to the grid.
 In addition, if row grouping or tree data is being used, the row reordering is also disabled.
 :::
 
-## 🚧 Row spanning
+## Row pinning [<span class="plan-pro"></span>](https://mui.com/store/items/material-ui-pro/)
+
+Pinned (or frozen, locked or floating) rows are rows that are visible at all times while the user scrolls the grid vertically.
```

Comment:
> Pinned (or frozen, locked or floating)

Btw, how do we add these different nomenclatures to Algolia?

Reply:
Not sure about that. I would expect some kind of index in the Algolia configuration.
Maybe @oliviertassinari can point out the direction?